requirements-dev.txt

```diff
@@ -1,3 +1,7 @@
 --editable .[dev]
 idaes-pse @ git+https://github.com/IDAES/idaes-pse@main
-pytest-icdiff >= 0.7  # readable dict diffs for test_flowsheet and others
\ No newline at end of file
+pydantic~=2.0
+fastapi
```

Comment:
should we add version numbers for both Fastapi and Uvicorn

Reply:
not unless we have to